config/routes.rb

```diff
@@ -70,6 +70,8 @@
   get 'pricing', to: 'pages#pricing'
   get 'internship', to: 'pages#jobs'
 
+  get 'blog', to: redirect('@cindy/tags/jelly-blog')
```

Comment:
this should route to a specific action where we can render the collection, probably just pages#blog for now.

Reply:
@dml-1 could you start using conventional comments in reviews? https://conventionalcomments.org/

Reply:
i am not entirely sure what you mean, but i will try something and push up a change!